README.md

```diff
@@ -41,6 +41,25 @@ The following bounties are available for anyone to pick up:
 
 <br>
 
+# Bounty Submission
+
+## Outside of [XRPLF](https://github.com/XRPLF)
```

Comment:
We should start with a positive conditional instead of `not` ______ because it can get a bit confusing. 

I'd recommend changing this to: 
```suggestion
## Contributing to an existing open source project
```

And including the relevant instructions

Reply:
(For the other title I'd recommend `## Making a new repository for your work` with these instructions)